book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,120 @@
+(ch-coworking-dropins)=
+
+# Drop-in Sessions
+
+Drop-in sessions are an informal collaboration format where a team or group of people host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.
+
+These sessions typically include an online component, typically using [Zoom](https://www.zoom.us/). 
+They may follow a different format or sign up person, depending on the needs and preferences of the organising team and the target community the sessions are designed for.
```

Comment:
```suggestion
They may follow a different format, depending on the needs and preferences of the organising team and the target community the sessions are designed for.
```
Or please explain what signup person means?

Reply:
Thank you, it was a typo!